guides.Rmd

```diff
@@ -196,136 +317,260 @@ Note the use of `%o%` to quickly generate the multiplication table, and that the
 
 
 
+## Scale guides {#scale-guide}
 
-### Dates: A special case {#date-scales}
+The previous sections describe features of ggplot2 that are the same for both axes and legend, but because legends are more complex than axes, there are additional topics to discuss that pertain only (or mostly) to legends. ggplot2 is designed so that in most cases the default legends will be sensible, but even so it is often necessary to override the defaults. You can do this using the `guide` argument of the corresponding scale function, or more conveniently, the `guides()` helper function. 
```

Comment:
This is no longer true since we have `guide_axis()` now. So maybe reframe to something like "scales allow you to control the high-level details of the guides, but if you want to control the details, you'll need to use the guide functions"